### DISCUSSION TITLE
Should I put on 90DNS before injecting (without AMS/SXOS) to avoid a ban?

### DISCUSSION THREAD

**Discussion by FieryMewtwo** [suchmememanyskill/TegraExplorer#62] [Q&A]
﻿

**Comment by suchmememanyskill** [comment]
what does this have to do with tegraexplorer?

**Reply by FieryMewtwo** [reply]
I mean, do I need to put on 90DNS before running TegraExplorer? I don't have CFW yet , and was wondering if I needed to take anti-ban precautions

**Reply by suchmememanyskill** [reply]
Running a payload has no correlation with bans. So no, running tegraexplorer will not get you banned (just like literally any other payload)

**Reply by FieryMewtwo** [reply]
Okay, sorry to bother you.
